ia/src/models.py

```
@@ -0,0 +1,33 @@
+#     def __init__(self, message, encrypt, received):
+#         """
+#         Initializes a iaParser body.
+#
+#         Args:
+#             message (list:str): list of messages
+#             encrypt (list:str): name of the team
+#             received (list:str): name of the machine
+#         """
+#         self.message = message
+#         self.encrypt = encrypt
+#         self.received = received
```

Pylint (3.3.7)

[convention] Final newline missing

(C0304)